SIL.Windows.Forms.DblBundle/SelectProjectDlgBase.cs

```diff
@@ -18,6 +18,11 @@ public abstract class SelectProjectDlgBase : IDisposable
 		protected SelectProjectDlgBase(bool allowProjectFiles = true, string defaultFile = null)
 		{
 			FileName = File.Exists(defaultFile) ? Path.GetFileName(defaultFile) : null;
+			
+		}
+
+		protected virtual OpenFileDialog CreateFileDialog()
+		{
```

Comment:
This fails to compile because `defaultFile` is unknown. We could store `defaultFile` (or `defaultDir`) in a private field.